apps/judicial-system/web-e2e/src/integration/Court/InvestigationCases/overview.spec.ts

```diff
@@ -60,6 +61,10 @@ describe(`${IC_OVERVIEW_ROUTE}/:id`, () => {
     intercept(caseDataAddition)
   })
 
+  it('should let the user know if the assigned defender has viewed the case', () => {
+    cy.getByTestid('alertMessage').should('not.match', ':empty')
```

Comment:
can we be more specific here with the alert message? Are we allowed to pass in a custom data-testid into the alert?

Reply:
There is currently no testid prop in the Alert component, but we could change that. How about adding an `id` prop that sets both an id and a data-testid? Or maybe just passing in a specific testid prop? What do you think?

Reply:
It would be nice to add the option so we know what `alertMessage` is being rendered. I think having a specific `data-testid` prop is the better option.